### PR TITLE
Auton GUI Improvements

### PR DESCRIPTION
**Pull request by jnnanni**
image

### CODE REVIEW

**Review by CameronTressler — APPROVED** [review]

File: base_station/gui/src/components/AutonTask.vue [hunk 1]
```diff
@@ -230,8 +229,6 @@ export default {
           this.RadioSignalStrength.signal_strength = msg.message.signal_strength.toFixed(1)
          }else if (msg.topic === '/autonomous') {
           this.Joystick = msg.message
-        } else if (msg.topic === '/rr_drop_complete') {
```
Comment (zwgold) [inline base_station/gui/src/components/AutonTask.vue:233]: Was there a reason for also including some removal of RR code in this PR? I thought you had removed all of it in an earlier one?

File: base_station/gui/src/components/AutonTask.vue [hunk 2]
```diff
@@ -62,15 +59,17 @@ import ArmControls from './ArmControls.vue'
...
 import Obstacle from './Obstacle.vue'
 import TargetList from './TargetList.vue'
 import DriveVelDataH from './DriveVelDataH.vue'
 import ZedGimbalAngles from './ZedGimbalAngles.vue'
 
-let interval;
+const navBlue = "#4695FF"
+const navGreen = "yellowgreen"
+const navRed = "lightcoral"
+const navGrey = "lightgrey"
```
Comment (CameronTressler) [inline base_station/gui/src/components/AutonTask.vue:72]: Change colors to constants as requested

File: base_station/gui/src/components/WaypointEditor_Auton.vue [hunk 3]
```diff
@@ -24,11 +24,16 @@
...
+          <button v-on:click="addWaypoint(input)">Add Waypoint</button>
+          <button v-on:click="addWaypoint(formatted_odom)">Drop Waypoint</button>
+        </div>
       </div>
       <div class="box1">
-        <h4 class="waypoint-headers">All Waypoints</h4>
+        <div class="all-waypoints">
+          <h3>All Waypoints</h3>
+          <button v-on:click="clearWaypoint">Clear Waypoints</button>
+        </div>
```
Comment (CameronTressler) [inline base_station/gui/src/components/WaypointEditor_Auton.vue:36]: Add Clear Waypoint button

**Comment by CameronTressler** [comment]
There seem to be git issues between this PR and auton-integration, will make a new PR on a new branch

**Pull request closed without merging**